internal/pkg/tfconf/module.go

```diff
@@ -187,3 +198,24 @@ func loadProviders(requiredProviders map[string]*tfconfig.ProviderRequirement, r
 	}
 	return providers
 }
+
+func loadOutputValues(options *Options) (map[string]*TerraformOutput, error) {
+	if options.OutputValuesPath == "terraform-outputs.json" {
+		_, err := exec.Command("cd", options.Path, "&&", "terraform", "output", "--json", ">>", options.OutputValuesPath).Output()
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
```

Comment:
This is the only remaining items, from where I can see. I'm gonna test this locally today. But my initial thought at the moment is that we actually should not set a default value for `--output-values-from`, but let me gather my thoughts on it first. Will sync up with you shortly.

Reply:
I've had some problem with this to make it work, but here is the working snippet

```go
func loadOutputValues(options *Options) (map[string]*TerraformOutput, error) {
	var out []byte
	var err error

	if options.OutputValuesPath == "" {
		cmd := exec.Command("terraform", "output", "-json")
		cmd.Dir = options.Path
		if out, err = cmd.Output(); err != nil {
			return nil, fmt.Errorf("caught error while reading the terraform outputs: %v", err)
		}
	} else {
		if out, err = ioutil.ReadFile(options.OutputValuesPath); err != nil {
			return nil, fmt.Errorf("caught error while reading the terraform outputs file at %s: %v", options.OutputValuesPath, err)
		}
	}

	var terraformOutputs map[string]*TerraformOutput
	err = json.Unmarshal(out, &terraformOutputs)
	if err != nil {
		return nil, err
	}
	return terraformOutputs, err
}
```

As you can see I'm not using `options.OutputValuesPath` at all unless it was explicitly passed by the user, and as such I suggest to remove the default value for it in `cmd/root.go` as well.

Reply:
I'll make that change this morning!